earth_enterprise/rpms/build.gradle

```diff
@@ -402,16 +402,64 @@ if (project.hasProperty('buildOpenGee')) {
     openGeeCommonRpm.dependsOn stageOpenGeeInstall
 }
 
+task openGeeCommonDeb (type: GeeDeb, dependsOn: openGeeSharedFiles) {
```

Comment:
Can you update the copyright year in the header to 2017, 2018?

Reply:
Sure. Should it be "2017, 2018" or "2017-2018"? Or does it matter either way?